CONTRIBUTING.md

```diff
@@ -28,9 +28,10 @@ All of the above also have `build` versions (e.g. `npm run build:js` or `npm run
 
 Please install an ESLint plugin for your editor. There is an `.eslintrc.json` file in the repo which encodes most of the coding style of the project.
 
+To lint and format the project, run `npm run lint`.
+
 Here are a few other guidelines that cannot be enforced via ESLint:
 
-- When you define a function, use a space between the opening paren and its name. That way we can search for "functionName (" and find its definition immediately, instead of having to wade through calls to the function.
 - Prefer single-word names over multi-word names. 3+ word names are especially frowned upon.
```

Comment:
Why are we removing this?

Reply:
@LeaVerou I removed this note because I added an [ESLint rule](https://github.com/LeaVerou/color.js/pull/238/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR35) to enforce this convention -- so I thought the note is no longer applicable. If you prefer, I'm happy to add it back.

Reply:
No need :) Thanks for the explanation!